impeller/display_list/dl_dispatcher.cc

```diff
@@ -778,7 +778,7 @@ void DlDispatcher::drawOval(const SkRect& bounds) {
                     .AddOval(skia_conversions::ToRect(bounds))
                     .SetConvexity(Convexity::kConvex)
                     .TakePath();
-    canvas_.DrawPath(path, paint_);
+    canvas_.DrawPath(std::move(path), paint_);
```

Comment:
I'm guessing the linter isn't helping us here if we miss the std::move too?

Reply:
If we don't move its actually a compile error!

Reply:
oh right, nice!